lib/glific/contacts/import.ex

```diff
@@ -204,6 +249,18 @@ defmodule Glific.Contacts.Import do
     end
   end
 
+  ## Only glific admin will be able to delete the contacts
+  @spec should_delete_contact?(User.t()) :: boolean()
```

Comment:
Let's create a single function name like 
```
can_take_action(:optin, other_params)
can_take_action(:delete, other_params)
```
it will make it more clear.

Reply:
I have used it directly in the function. doesn't make sense to create another function just for one check